drivers/counter/counter_qmsi_aonpt.c

```diff
@@ -108,8 +109,9 @@ static u32_t aon_timer_qmsi_read(struct device *dev)
 }
 
 static int aon_timer_qmsi_set_alarm(struct device *dev,
-				    counter_callback_t callback,
-				    u32_t count, void *user_data)
+				    u32_t ticks,
+				    counter_wrap_callback_t callback,
+				    void *user_data)
```

Comment:
note that new api does not require to set wrap only when counter is already started. So `if (QM_AONC[QM_AONC_0]->aonpt_cfg == 0)` can be removed.  Basically, it should be possible to setup the counter and then start it.

Reply:
done